src/remarx/polar_utils.py

```diff
@@ -0,0 +1,118 @@
+import csv
+from collections import defaultdict
+
+import polars as pl
+
+
+def load_title_phrases(filename: str) -> dict[str, str]:
```

Comment:
The value of the returned dictionary is a list of strings, isn't it?

```suggestion
def load_title_phrases(filename: str) -> dict[str, list[str]]:
```

Reply:
Might be useful to set a default filename of `"data/title_searchphrases.csv"`, since we include that file in the repo and you're assuming those titles in the sentence preparation code below